cmake/CMakeLists.txt

```diff
@@ -13,17 +13,6 @@ if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.20")
   cmake_policy(SET CMP0117 NEW)
 endif()
 
-# Support OS X versions 10.12+
-# This variable is ignored on non-Apple platforms and needs to be set prior to the first project(...) invocation
-# TODO: Make the miniumum deployment target MacOSX version configurable
-# If the CMAKE_OSX_DEPLOYMENT_TARGET was set (currently valid only for iOS build),
-# we will cache its value instead of the default "10.12"
-if (NOT DEFINED CMAKE_OSX_DEPLOYMENT_TARGET)
```

Comment:
If someone is building from source on 10.15 for example, doesn't the minimum deployment target then become 10.15 by default ? We are setting the env var in our packaging pipeline, but should we also have this defaulting logic (to10.14) in build.py ?

Reply:
I think it only impact who provide onnxruntime packages for the others. If a user build and run onnxruntime on the same machine, that's fine. If he/she wants to distribute the package, usually he/she knows how to do it and set this env properly. The env var is well documented on Apple and cmake's website. Apple developers should be familiar with it.